addons/mk6mortar/CfgVehicles.hpp

```diff
@@ -29,6 +29,7 @@ class CfgVehicles {
         class ace_csw {
             proxyWeapon = QFUNC(csw_getProxyWeapon);
             magazineLocation = "_target selectionPosition 'usti hlavne'";
+            container = "ACE_Box_82mm_Mo_Combo";
```

Comment:
I'm not a fan of this, why did you want to add this?

Reply:
Mortars spawning with an ammo box that looks nice (instead of a generic .50 BMG ammo can. The container's inventory is cleaned up on init, only the CSW's ammo is added to it.

Reply:
Okay yeah fair